src/eval_context.rs

```diff
@@ -1367,13 +1367,13 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
         if let Lvalue::Local { frame, local, field } = lvalue {
             let mut allocs = Vec::new();
             let mut msg = format!("{:?}", local);
+            if let Some(field) = field {
+                write!(msg, ".{}", field).unwrap();
```

Comment:
This failed to compile, and the error message makes me realize it should only be printing the index part of the `field` tuple.